modules/docs/mdx/7.0-MIGRATION-GUIDE.mdx

```diff
@@ -36,244 +36,262 @@ from Canvas Kit v6 to v7:
 
-### Remove `IconButton` `plain`, `square`, and `squareFilled` Variants
+> Note: See below for more information about how to manually migrate from
+> [unsupported v6 variants](#unsupported-iconbutton-variants).
 
-As part of simplifying our variants and cleaning up the API, we've removed the variants `square`,
-`squareFilled` and `plain`. We've also removed the prop `toggled`.
+🤖 Use the codemod to migrate your `IconButton` components in v6 to their corresponding `Button` in
+v7:
 
-Plain, Square, or Square-Filled Icon Buttons should either transition to the new icon only Primary,
-Secondary, and Tertiary button variants, the Toolbar Icon Button, or the Segmented Control component
+```tsx
+// v6
+<IconButton icon={plusIcon} />
+<IconButton variant="circleFilled" icon={plusIcon} />
+<IconButton variant="inverse" icon={plusIcon} />
+<IconButton variant="inverseFilled" icon={plusIcon} />
 
-If you would still like to have a squared icon, you can use
-[ToolbarIconButton](https://workday.github.io/canvas-kit/?path=/docs/components-buttons-button-react-toolbar--toolbar-icon-button)
-which also has the `toggled`prop.
+// v7
+<TertiaryButton icon={plusIcon} />
+<SecondaryButton icon={plusIcon} />
+<TertiaryButton variant="inverse" icon={plusIcon} />
+<SecondaryButton variant="inverse" icon={plusIcon} />
+```
 
-If this button doesn't fit your specific use case, we've introduced a low level
-[`BaseButton` component](#basebutton-component). This component allows you to modify the colors,
-props, and size of the button. For an example of this being used, you can reference our
-[PageButton component](https://github.com/Workday/canvas-kit/blob/master/modules/react/pagination/lib/Pagination/PageButton.tsx).
+##### Unsupported IconButton Variants
 
-### Rendering an icon button
+In order to simplify our API, we no longer support the `plain`, `square`, and `squareFilled`
+variants from `IconButton`.
 
-In order to render a icon button, omit any `children` and pass the button an `icon` prop.
+`square`, `squareFilled` and `plain` `IconButton` components should be transitioned to the new
+icon-only versions of the `Button` components (`PrimaryButton`, `SecondaryButton`, and
+`TertiaryButton`), `ToolbarIconButton`, or `SegmentedControl`. You may use `ToolbarIconButton` if
+you would still like to have a square icon.
 
-Example
+These changes are **not** handled by the codemod.
 
-v6
+> If none of these options support your use case, we've introduced a low-level
+> [`BaseButton`](#basebutton) component which allows you to modify the colors, props, and size of
+> the button. For an example of it being used, reference our
+> [`PageButton`](https://github.com/Workday/canvas-kit/blob/master/modules/react/pagination/lib/Pagination/PageButton.tsx)
+> component.
 
-```tsx
-<IconButton variant="circleFilled" icon={plusIcon} />
-```
+##### Removal of IconButtonProps
 
-v7
+The `IconButtonProps` interface no longer exists now that `IconButton` has been removed.
```

Comment:
The codmod also doesn't handle removing this import i believe

Reply:
Cool, I mention right below (on L212) that the codemod doesn't handle the removal of `IconButtonProps`. Or do you think we should be more clear about the codemod not removing the import specifically.

Reply:
nope nope hadn't seen the lines below, thanks!